pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -124,6 +131,13 @@ func (q *QueryReconciler) runListRestorePoints(ctx context.Context, _ *ctrl.Requ
 		return err
 	}
 
+	// update query message after calling vclusterops API successfully
+	err = vrpqstatus.UpdateMessageStatus(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
+		"Queried successful")
```

Comment:
I know I had suggested "Queried successful" :smile: , but seeing its use I wonder if "Query successful" would be better?